process-document/src/main/java/com/ritense/processdocument/service/impl/CamundaProcessJsonSchemaDocumentAssociationService.java

```diff
@@ -126,8 +144,26 @@ public CamundaProcessJsonSchemaDocumentDefinition getProcessDocumentDefinition(P
     public List<CamundaProcessJsonSchemaDocumentDefinition> findProcessDocumentDefinitions(String documentDefinitionName) {
         // TODO: (VIEW JsonSchemaDocument) / (ADMIN role, so separate endpoint)
 
-        return processDocumentDefinitionRepository
+        List<CamundaProcessJsonSchemaDocumentDefinition> results = processDocumentDefinitionRepository
```

Comment:
```suggestion
        var results = processDocumentDefinitionRepository
```

Reply:
Just a different flavor, so I'm not going to apply this. I would argue var is more difficult to read.